modules/representation_management/app/controllers/representation_management/v0/pdf_generator_2122_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    class PdfGenerator2122Controller < RepresentationManagement::V0::PdfGeneratorBaseController
+      def create
+        form = RepresentationManagement::Form2122Data.new(flatten_form_params(form_params))
```

Comment:
You don't need to pass `form_params` into `flatten_form_params`, instead you can do this:

```
def flatten_form_params
  {
    record_consent: form_params[:record_consent],
    consent_limits: form_params[:consent_limits],
    consent_address_change: form_params[:consent_address_change],
    conditions_of_appointment: form_params[:conditions_of_appointment]
  }.merge(flatten_veteran_params(form_params))
    .merge(flatten_claimant_params(form_params))
    .merge(flatten_representative_params(form_params))
end
```

Reply:
Done.